src/SessionOne.java

```diff
@@ -5,50 +5,73 @@ public static void main(String[] args) {
         Scanner input = new Scanner(System.in);
         System.out.println("Press 1, if you want to see if it's an SessionOne number");
```

Comment:
That´s was a mistake, I replaced the class name.

Reply:
That's ok, we all do mistakes, that's why code reviews exist, don't think about code reviews as people trying to criticise your work, it's a collective effort and you should think of it like that, sometimes it might seem like people is being rude, but it's not like it, people are just asking what you meant, since you are not there to explain. Never, never, never take code reviews personal, (I learned that the hard way) take them as an opportunity to improve and to work as a team. Be grateful at people taking a look at your work, they are dedicating time to you. Dedicate time to them as well. Embrace code reviews.